docs/swagger.yml

```diff
@@ -3,7 +3,7 @@ swagger: '2.0'
 info:
   title: argo-api-authN
   description: "The purpose of the Authentication Service is to provide the ability to different services to use alternative authentication mechanisms without having to store additional user info or implement new functionalities.The AUTH service holds various information about a service’s users, hosts, API urls, etc, and leverages them to provide its function."
-  version: ""
+  version: "v1"
```

Comment:
versioning prefix needs to be added also as a base_path in swagger

Reply:
added